network/common/src/main/java/org/apache/spark/network/TransportContext.java

```diff
@@ -55,14 +56,14 @@
   private final Logger logger = LoggerFactory.getLogger(TransportContext.class);
 
   private final TransportConf conf;
-  private final RpcHandler rpcHandler;
+  private final RpcHandler appRpcHandler;
```

Comment:
Ah -- I see, this is the guy before the bootstraps are executed, which is used by the client directly and as the original handler otherwise.